hstio/hstio.c

```diff
@@ -1074,7 +1071,10 @@ void copyOffsetSingleGroup(SingleGroup * output, const SingleGroup * input,
     if (!output || !input)
         return;
     //WARNING - assumes row major storage
-    assert(output->sci.data.storageOrder == ROWMAJOR && output->sci.data.storageOrder == ROWMAJOR);
+    if (output->sci.data.storageOrder != ROWMAJOR ||
+            input->sci.data.storageOrder != ROWMAJOR) {
+        return;
```

Comment:
The `assert` was grave enough end the program `.storageOrder` wasn't `ROWMAJOR`. We may want to consider setting `status` before returning.

Reply:
OK I set the status on L1076. But everywhere else in this file, the function declare local `int status = 0;` and then either return the status or pass it by address into something else. Can you please double check that my addition here is okay? Thanks!

Reply:
I refreshed my memory a bit.  It looks like the _assert_ business started when JN implemented the CTE Version 2 - **argh**.  **Even more argh** when I did not recode the asserts when I updated to CTE Version 3.

It looks like the status will bubble up the chain and finally deliver the status (doing this quickly so may not be exactly this chain):
dopcte-gen3.c
           copyOffsetSingleGroup
         extractAmp
       doPCTEGen3
docte.c
        DoCTE
acscte.c
        ACScte
calacs.c
        ProcessACSCCD
     CalAcsRun

Please just force (hardcode for the test) the status to failure, process an image, and see what pops to make sure all is well.